data/mods/MindOverMatter/effectoncondition/eoc_nether_attunement_events.json

```
@@ -644,20 +657,44 @@
         "run_eocs": [
           {
             "id": "EOC_DRAIN_EFFECT_CHECK_HEALTH_CHANGE_2",
-            "condition": { "math": [ "rand(1) >= 1" ] },
+            "condition": { "math": [ "rand(2) >= 2" ] },
             "effect": [
-              { "u_add_effect": "effect_nether_attunement_health_bonus", "duration": "30 seconds" },
+              { "math": [ "u_health() -= (rand(10) + 5)" ] },
```

```suggestion
              { "math": [ "u_health() += (rand(10) + 5)" ] },
```
Given the messages these are associated with, I'm pretty sure these signs are supposed to be the other way around